ddl/ddl_api.go

```diff
@@ -5400,6 +5403,9 @@ func (d *ddl) CreateIndex(ctx sessionctx.Context, ti ast.Ident, keyType ast.Inde
 		return errors.Trace(err)
 	}
 
+	if t.Meta().TableCacheStatusType != model.TableCacheStatusDisable {
```

Comment:
I think we should do checking both before and in schedule.

Reply:
you mean is double check?